src/mqtt-broker/src/subscribe/subscribe_manager.rs

```diff
@@ -228,6 +236,79 @@ impl SubscribeManager {
         }
     }
 
+    pub async fn save_exclusive_subscribe(
+        &self,
+        subscribe: Subscribe,
+    ) -> Result<Option<SubscribeReasonCode>, CommonError> {
+        for filter in subscribe.filters.clone() {
+            if !topic_util::is_exclusive_sub(&filter.path) {
+                continue;
+            }
+            if self
+                .metadata_cache
+                .get_cluster_info()
+                .feature
+                .exclusive_subscription_available
+                == AvailableFlag::Disable
+            {
+                return Ok(Some(SubscribeReasonCode::ExclusiveSubscriptionDisabled));
+            }
+            let topic_name = topic_util::decode_exclusive_sub_path_to_topic_name(&filter.path);
+            if !set_nx_exclusive_topic(self.client_pool.clone(), topic_name.to_owned())
+                .await?
+                .success
+            {
+                return Ok(Some(SubscribeReasonCode::TopicSubscribed));
+            }
+        }
+        Ok(None)
+    }
+
+    pub async fn remove_exclusive_subscribe(
+        &self,
+        un_subscribe: Unsubscribe,
+    ) -> Result<(), CommonError> {
+        for filter in un_subscribe.filters.clone() {
+            self.remove_exclusive_subscribe_by_sub_path(filter.clone())
+                .await?;
+        }
+        Ok(())
+    }
+
+    pub async fn remove_exclusive_subscribe_by_client_id(
+        &self,
+        client_id: &str,
+    ) -> Result<(), CommonError> {
+        for (_, subscriber) in self.exclusive_subscribe.clone() {
+            if subscriber.client_id == *client_id {
+                self.remove_exclusive_subscribe_by_sub_path(subscriber.sub_path)
+                    .await?;
+            }
+        }
+        Ok(())
+    }
+
+    async fn remove_exclusive_subscribe_by_sub_path(
```

Comment:
As above， try: 
async fn remove_exclusive_subscribe_by_sub_path(
        &self,
        un_sub_path: &str,
    )

Reply:
okay , i'll make this change